velox/expression/tests/ExpressionVerifier.cpp

```diff
@@ -232,64 +278,74 @@ bool ExpressionVerifier::verify(
 
 void ExpressionVerifier::persistReproInfo(
     const VectorPtr& inputVector,
+    std::vector<column_index_t> columnsToWarpInLazy,
```

Comment:
do we need to push down this vector here? Couldn't we infer it based on which column in inputVector have `isLazy()` set?